src/app.js

```diff
@@ -1,21 +1,37 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
+const http = require('http');
+
+const PORT = process.env.PORT || 3000;
+
+function createQuery(string) {
+  const paramsEntries = string
+    .split('&')
+    .map(param => param.split('='))
+    .filter(param => param[0]);
+
+  return Object.fromEntries(paramsEntries);
```

Comment:
Prefer using [URLSearchParams](https://developer.mozilla.org/uk/docs/Web/API/URLSearchParams) to custom parsing. This will fail on invalid params, eg `?test==test2`